fvm/src/machine/mod.rs

```diff
@@ -220,11 +224,6 @@ pub struct MachineContext {
     /// The initial state root on which this block is based.
     pub initial_state_root: Cid,
 
-    /// The base fee that's in effect when the Machine runs.
-    ///
-    /// Default: 0.
-    pub base_fee: TokenAmount,
-
```

Comment:
There's more refactor/consolidation we'll need to do here between `NetworkConfig` and `NetworkContext`.

`NetworkConfig` is confusing because it actually contains **machine** configuration:

- max call depth
- max wasm stack
- actor debugging
- actor redirects
- builtin overrides

As well as some network attributes which should live in `NetworkContext`:

- network version
- price list

But we don't need to do it _now_.